README.md

```diff
@@ -158,6 +158,12 @@ folder of your own test data set.
 PS.: Some unit tests cannot be executed when the offline test data set is in use, those are automatically marked as
 skipped.
 
+## Our Backward Compatibility (BC) Promise
+This library follows the [Semantic Versioning](https://semver.org) strategy. It means only major releases (such as 3.0, 4.0 etc.) are allowed to break backward compatibility and we do our best to keep it this way.
+The PHP API Client has to be compatible with the latest versions of the supported Apigee Edge APIs.
```

Comment:
You said "has to be compatible", but I think you meant to declare that we will support the latest version?  See if this is correct:
```suggestion
The PHP API Client will always be compatible with the latest versions of the supported Apigee Edge APIs.
```